source/jormungandr/jormungandr/interfaces/v1/serializer/fields.py

```diff
@@ -192,8 +194,15 @@ def to_value(self, value):
         coords = []
         if value.type == response_pb2.STREET_NETWORK:
             coords = value.street_network.coordinates
-        elif value.type == response_pb2.CROW_FLY and len(value.shape) != 0:
-            coords = value.shape
+        elif value.type == response_pb2.CROW_FLY:
+            if len(value.shape) != 0:
+                coords = value.shape
+            else:
+                try:
+                    coords.append(get_pt_object_coord(value.origin))
+                    coords.append(get_pt_object_coord(value.destination))
+                except UnableToParse:
```

Comment:
The exception can also be [InvalidArgument](https://github.com/CanalTP/navitia/blob/261199b377956a211423300a6333ad539fe3d9a7/source/jormungandr/jormungandr/utils.py#L391) No?